.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]

```

Comment:
use one file setup.cfg for all settings example: https://pycqa.github.io/isort/docs/configuration/config_files.html

https://github.com/Safintim/django-blog/blob/master/setup.cfg